{"nick":"isaacs","message":"JasonSmith: Hey check this out: https://gist.github.com/3068840","date":"2012-07-08T01:08:12.426Z","type":"message"}
{"nick":"isaacs","message":"JasonSmith: any idea what's up with that?  happens when trying to PUT to my _users doc","date":"2012-07-08T01:08:25.152Z","type":"message"}
{"nick":"isaacs","message":"(signed in)","date":"2012-07-08T01:08:30.720Z","type":"message"}
{"nick":"josdeha_rt","date":"2012-07-08T13:04:58.355Z","type":"join"}
{"nick":"josdeha_rt","reason":"Remote host closed the connection","date":"2012-07-08T13:54:36.924Z","type":"quit"}
{"nick":"tilgovi","date":"2012-07-08T22:40:27.757Z","type":"join"}
{"nick":"JasonSmith","message":"isaacs: I believe that is a bug in CouchDB saying the validate_doc_update is throwing an exception","date":"2012-07-08T23:43:05.394Z","type":"message"}
